<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Spawn</class>
 <widget class="QWidget" name="Spawn">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>460</width>
    <height>319</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Spawn</string>
  </property>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>140</y>
     <width>331</width>
     <height>41</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QComboBox" name="comboBox">
      <item>
       <property name="text">
        <string>PushButton</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>LineEdit</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Label</string>
       </property>
      </item>
     </widget>
    </item>
    <item>
     <widget class="QSpinBox" name="spinBox">
      <property name="maximum">
       <number>5</number>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton">
      <property name="text">
       <string>Spawn!</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
